src/js/edu-benefits/components/military-history/MilitaryServiceTour.jsx

```diff
@@ -29,23 +30,19 @@ export default class MilitaryServiceTour extends React.Component {
               name="serviceStatus"
               field={tour.serviceStatus}
               onValueChange={(update) => {onValueChange('serviceStatus', update);}}/>
-          <DateInput required
-              errorMessage="Please provide a valid date"
-              validation={validateIfDirtyDateObj(tour.dateRange.from, isValidDateField)}
+          <ErrorableCurrentOrPastDate required
               label="Start of service period:"
               name="fromDate"
-              day={tour.dateRange.from.day}
-              month={tour.dateRange.from.month}
-              year={tour.dateRange.from.year}
+              date={tour.dateRange.from}
               onValueChange={(update) => {onValueChange('dateRange.from', update);}}/>
-          <DateInput required
-              errorMessage={isValidDateRange(tour.dateRange.from, tour.dateRange.to) ? 'Please provide a valid date' : 'Date separated must be after date entered'}
-              validation={validateIfDirtyDateObj(tour.dateRange.to, date => isValidDateRange(tour.dateRange.from, date))}
-              label="End of service period"
+          <ErrorableDate
+              validation={{
```

Comment:
@akainic @alexose I changed how the validation works here. I think this is pretty simple. I also made it so you could pass in an array, so we can do multiple validations with their own messages more easily.